photutils/detection/deblend.py

```diff
@@ -112,7 +112,7 @@ def deblend_sources(data, segment_img, npixels, filter_kernel=None,
         labels = segment_img.labels
     labels = np.atleast_1d(labels)
 
-    data = _convolve_data(data, filter_kernel, mode='constant',
+    data = filter_data(data, filter_kernel, mode='constant',
                           fill_value=0.0)
```

Comment:
The indentation on this line needs to be adjusted.  The "f" in `fill_value` should be directly below the "d" in `data` in the line above.